addons/editor-number-arrow-keys/userscript.js

```diff
@@ -116,30 +109,11 @@ export default async function ({ addon }) {
     // number input (increase or decrease by 1). If we didn't prevent, the user would be increasing twice.
 
     let changeBy = e.code === "ArrowUp" ? 1 : -1;
-    if (addon.settings.get("useCustom")) {
-      let settingValue = e.shiftKey
-        ? addon.settings.get("shiftCustom")
-        : e.altKey
-        ? addon.settings.get("altCustom")
-        : addon.settings.get("regularCustom");
-      if (settingValue === "") settingValue = 0;
-      let valueAsFloat = parseFloat(settingValue);
-      if (valueAsFloat < 0) valueAsFloat *= -1; // If user typed a negative number, we make it positive
-      if (Number.isNaN(valueAsFloat)) {
-        return;
-      } else if (valueAsFloat === 0 || (valueAsFloat < 100000000 && valueAsFloat > 0.00000099)) {
```

Comment:
This excluded numbers that were too precise back when the settings were string inputs. Not sure if we still want to do something about that. This safeguard helps prevent accidental precision loss when incrementing by precise values.

Reply:
I assume #6794 will be merged at the same time as this?

Reply:
> I assume #6794 will be merged at the same time as this?

Well, that PR doesn't change the precision of the setting values, but the number inputs seem to automatically truncate them, and using scientific notation makes incrementing throw an error because it can't be converted to BigInt, which just means nothing happens.